packages/voorbeeld-design-tokens/figma/voorbeeld.tokens.json

```
@@ -3841,6 +3841,20 @@
       }
     }
   },
+  "components/icon": {
+    "utrecht": {
+      "icon": {
+        "color": {
+          "$type": "color",
+          "$value": "{utrecht.document.color}"
+        },
+        "size": {
+          "$type": "sizing",
+          "$value": "{voorbeeld.size.xs}"
+        }
```

```suggestion
      "icon": {
        "size": {
          "$type": "sizing",
          "$value": "{voorbeeld.size.xs}"
        }
```

Color moet niet ingesteld zijn